src/components/button/LogoutButton.jsx

```diff
@@ -3,25 +3,24 @@ import { useHistory } from "react-router-dom";
 
 import "./button.scss";
 import useToastContext from "../../hooks/useToastContext";
+import useUserContext from "../../hooks/useUserContext";
 
 const LogoutButton = () => {
   const history = useHistory();
   const setToast = useToastContext();
+  const { setIsUserAuthenticated } = useUserContext();
 
-  return (
-    <IonButton
-      onClick={() => {
-        localStorage.removeItem("accessToken");
-        localStorage.removeItem("refreshToken");
-        localStorage.removeItem("userId");
-        localStorage.removeItem("isGuest");
-        history.replace("/");
-        setToast({ message: "Logged out successfully.", color: "success" });
-      }}
-    >
-      Log Out
-    </IonButton>
-  );
+  const logoutUser = () => {
+    localStorage.removeItem("accessToken");
+    localStorage.removeItem("refreshToken");
+    localStorage.removeItem("userId");
+    localStorage.removeItem("isGuest");
+    history.replace("/");
+    setToast({ message: "Logged out successfully.", color: "success" });
+    setIsUserAuthenticated(false);
+  };
```

Comment:
This can be placed under utils / services, because it can come in handy in other places if, for example, we need to force logout a user (e.g. invalid refresh token)